README.md

```diff
@@ -0,0 +1,22 @@
+# Alan Technologies, Inc - Homebrew formulae
+[Homebrew](https://brew.sh/) formulae for _alantech_ packages.
+
+## Usage
+Add new tap:
+```bash
+brew tap alantech/homebrew-core
+```
+
+Install:
+```bash
+brew install <formula>
+```
+
+Update:
+```bash
+brew update
+brew upgrade <formula>
+```
+
+## Available formulae
+- [anycloud](https://github.com/alantech/anycloud)
```

Comment:
Should we add `alan` to this, as well?

Reply:
I've added it with checkboxes since it is not implemented yet, what do you think?